src/Components/Cards/ReservationConfirmationDetailsCard/ReservationConfirmationDetailsCard.js

```diff
@@ -0,0 +1,74 @@
+//#region Import Neccessary Dependencies
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+//#endregion
+
+// Import Stylings
+import './ReservationConfirmationDetailsCard.css';
+
+//#region Import Icons
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faScrewdriverWrench } from '@fortawesome/free-solid-svg-icons';
+//#endregion
+
+// Define ReservationConfirmationDetailsCard Component
+function ReservationConfirmationDetailsCard(props) {
+
+  // Destructuring props
+  const { className, modelName, modelPhoto, typeName, availableCount, quantity } = props;
+
+  // State to handle model photo
+  const [equipmentModelPhoto, setEquipmentModelPhoto] = useState(modelPhoto);
+
+  // Singular or plural text based on available count
+  const itemText = availableCount > 1 ? 'items' : 'item';
+
+  return (
+    <div className={`ReservationConfirmationDetailsCard-GeneralContainer ${className}`}>
+      <div className='ReservationConfirmationDetailsCard-Container'>
+        <div className='ReservationConfirmationDetailsCard-ModelPhoto'>
+          {equipmentModelPhoto && (
+            <img src={modelPhoto}
```

Comment:
Same Sonarlint error as before.

Reply:
Oh yeah, it was because of the OnError function of img and img does not seen as button. However, there seems to not having the other ways to treat it as button so I think we will just go with it for now!